pkg/client/client_credentials_object_set.go

```diff
@@ -1,12 +1,23 @@
+<<<<<<< HEAD
 // Copyright 2020 Hewlett Packard Enterprise Development LP
+=======
+// Copyright 2021 Hewlett Packard Enterprise Development LP
+>>>>>>> 87138d62 (Add autogenerated files)
```

Comment:
Looks like an issue here. git rebase/merge seems to have a conflict

Reply:
Fixed! thanks!